game/dota_addons/dota_imba/scripts/vscripts/hero/hero_shadow_shaman.lua

```diff
@@ -30,33 +30,31 @@ function imba_shadow_shaman_mass_serpent_ward:OnSpellStart()
 
     -- Emit spawn particle
     local spawn_particle_fx = ParticleManager:CreateParticle(spawn_particle, PATTACH_ABSORIGIN, caster)
-    ParticleManager:SetParticleControl( spawn_particle_fx, 0, spawn_point )
-
-    -- Create 8 wards in a box formation (note the vectors)
-    ward_table  =   {}
-    ward_table[1] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(-64, -64 ,0), true, caster, caster, caster:GetTeamNumber())
-    ward_table[2] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(-64, 0 ,0), true, caster, caster, caster:GetTeamNumber())
-    ward_table[3] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(-64, 64 ,0), true, caster, caster, caster:GetTeamNumber())
-    ward_table[4] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(0, -64 ,0), true, caster, caster, caster:GetTeamNumber())
-    ward_table[5] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(0, 64 ,0), true, caster, caster, caster:GetTeamNumber())
-    ward_table[6] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(64, -64 ,0), true, caster, caster, caster:GetTeamNumber())
-    ward_table[7] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(64, 0 ,0), true, caster, caster, caster:GetTeamNumber())
-    ward_table[8] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(64, 64 ,0), true, caster, caster, caster:GetTeamNumber())
-    -- Create 2 additional wards next to the other ones
-    ward_table[9] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(-128 , 64 ,0), true, caster, caster, caster:GetTeamNumber())
-    ward_table[10] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(128 , 64 ,0), true, caster, caster, caster:GetTeamNumber())
-
-    -- #1 TALENT: More Serpent Wards
-    if caster:HasTalent("special_bonus_imba_shadow_shaman_1") then
-        ward_table[11] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(-128, -64 ,0), true, caster, caster, caster:GetTeamNumber())
-        ward_table[12] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(128, -64 ,0), true, caster, caster, caster:GetTeamNumber())
-        ward_table[13] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(192, 64 ,0), true, caster, caster, caster:GetTeamNumber())
-        ward_table[14] = CreateUnitByName(ward_name..ward_level, spawn_point + Vector(-192, 64 ,0), true, caster, caster, caster:GetTeamNumber())
-    end
+    ParticleManager:SetParticleControl( spawn_particle_fx, 0, target_point )
+
+	-- #1 TALENT: More Serpent Wards
+    if caster:HasTalent("special_bonus_imba_shadow_shaman_1") then 
+		ward_count = ward_count + caster:FindTalentValue("special_bonus_imba_shadow_shaman_1")
+	end
+	
+    -- Create 8 wards in a box formation (note the vectors) + more wards near them
+    local formation_vectors = {
+								Vector(-128 , 64 ,0), Vector(-64, 64, 0) ,  Vector(0, 64 ,0), Vector(64, 64 ,0) , Vector(128 , 64 ,0),
```

Comment:
I mean, I'm sure it looks different in your text editor, but here on GH the indentation is drastically different from the previous changes.

If you think it's fine then I won't push, just thought I'd point it out.